.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
please remove this file. We support multiple version of node.js.

Reply:
I was trying to prevent node 15/(npm 7.5.1?) from rewriting the package-lock to v2.

The .nvmrc just hints the version to developers working on the project, not users of the project, right?

We also have this

https://github.com/nearform/fastify-secrets-vault/blob/dd4fac46c8406df00b6751435b1c0048dc1512e5/package.json#L31-L33